framework/src/variables/MooseVariableFV.C

```diff
@@ -805,14 +793,13 @@ MooseVariableFV<OutputType>::evaluate(const FaceArg & face,
   mooseAssert(state == 0, "Only current time state supported.");
   const FaceInfo * const fi = face.fi;
   mooseAssert(fi, "The face information must be non-null");
-  if (isExtrapolatedBoundaryFace(*fi).first)
-    return getExtrapolatedBoundaryFaceValue(*fi);
-  else if (isInternalFace(*fi))
+  if (isInternalFace(*fi))
     return Moose::FV::interpolate(*this, face);
   else
   {
-    mooseAssert(isDirichletBoundaryFace(*fi), "We've run out of face types");
-    return getDirichletBoundaryFaceValue(*fi);
+    Moose::SingleSidedFaceArg ssfa = {
+        face.fi, face.limiter_type, face.elem_is_upwind, face.correct_skewness, face.elem_sub_id};
```

Comment:
As for the PBBLF, let's be sure that we are providing the correct subdomain ID to the SSF. E.g. let's make sure the variable is defined on the subdomain ID

Reply:
Added a conditional subdomain ID based on if the variable is defined on the element!